src/providers/scala.rs

```diff
@@ -76,6 +81,11 @@ impl ScalaProvider {
         pkgs
     }
 
+    pub fn get_run_dep_pkgs(&self) -> Vec<Pkg> {
```

Comment:
this function isn't used anywhere?

Reply:
ah I forgot to remove that. Was trying to see if I can use a nix image in the StartPhase and packages to it. let me remove it.